package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-calendly",
-  "version": "4.0.1",
+  "version": "4.0.2",
```

Comment:
Can you remove this change? I can update the version when I publish the changes to npm.